### PR TITLE
chore(docs): Remove non-inclusive language

### PR DESCRIPTION
**Pull request by noahsb**
## Description

- This is an attempt to replace/remove non-inclusive language found in the docs (#18702)
- Used Carolyn's wonderful [guide](https://dev.to/meeshkan/how-to-remove-condescending-language-from-documentation-4a5p) to do this (thank you Carolyn!)

## Additional Details

- Reviewed every warning in the docs to ensure there weren't any issues 
- Initial run of `npx alex` on docs resulted in **599** warnings - second run resulted in **493**
- The VAST majority of the remaining warnings arise from non-issue words such as _hooks, execute, host_, etc. 
- There are a handful of cases that require more extensive editing, such as the following example:
    in `sourcing-from-buttercms.md`, we say it's "Easy as Butter" - issues like these may require additional considerations

## Related Issues

Contributes to #18702 - while this fixes the vast majority of instances of non-inclusive language in the docs, there is still room for improvement

### CODE REVIEW

**Comment by carolstran** [comment]
I'm biased but FWIW I think this looks great 💜

**Review by okasi — APPROVED** [review]

File: docs/docs/plugins-themes-and-starters.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ In the Gatsby ecosystem, there's more than one way to build a site. To help you
 
 ## What is a plugin?
 
-Gatsby plugins are Node.js packages that implement Gatsby APIs and are commonly installed through a registry like npm. There are many types of [plugins](/plugins/), including data sourcing, SEO, responsive images, offline support, support for Sass, TypeScript, sitemaps and RSS, Google Analytics, and more. You can also [make your own plugins](/docs/creating-plugins/) and either distribute them for fellow Gatsby developers to use or [install them locally](/docs/loading-plugins-from-your-local-plugins-folder/).
+Gatsby plugins are Node.js packages that implement Gatsby APIs and are commonly installed through a registry like npm. There are many types of [plugins](/plugins/), including data sourcing, SEO, responsive images, offline support, support for Sass, TypeScript, sitemaps and RSS, Google Analytics, and more. You can also [make your own plugins](/docs/creating-plugins/) and either distribute them for Gatsby developers to use or [install them locally](/docs/loading-plugins-from-your-local-plugins-folder/).
```
Comment (ascorbic) [inline docs/docs/plugins-themes-and-starters.md:9]: I think "fellow" is ok in the adjectival sense

Comment (noahsburroughs) [reply docs/docs/plugins-themes-and-starters.md:9]: You make a good point. I saw the warning for this and didn't realize it was referencing exclusively the noun. I'll revert that change. (big fan of your work btw @ascorbic)

**Comment by noahsb** [comment]
Hi, I implemented the suggested changes a couple weeks ago - could this please be re-reviewed?

**Review by LekoArts — DISMISSED** [review]
Thank you! 👍  There seem to be some merge conflicts, can you fix those? Then we can get this in!

**Review by LekoArts — APPROVED** [review]
Thank you! 👍

**Pull request merged**